### ISSUE TITLE
Bugs in Quick Start notebooks

### ISSUE THREAD

**Issue by MitchellAcoustics** [MitchellAcoustics/Soundscapy#44]

**Comment by MitchellAcoustics** [comment]
```
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
c:\Users\mitch\Documents\GitHub\Soundscapy\examples\QuickStart.ipynb Cell 7 in <cell line: 1>()
----> [1](vscode-notebook-cell:/c%3A/Users/mitch/Documents/GitHub/Soundscapy/examples/QuickStart.ipynb#ch0000006?line=0) df, excl = df.isd.validate_dataset(allow_na=False)
      [2](vscode-notebook-cell:/c%3A/Users/mitch/Documents/GitHub/Soundscapy/examples/QuickStart.ipynb#ch0000006?line=1) df

File ~\Documents\GitHub\Soundscapy\soundscapy\isd.py:92, in ISDAccessor.validate_dataset(self, paq_aliases, allow_lockdown, allow_na, verbose, val_range)
     84 def validate_dataset(
     85     self,
     86     paq_aliases=None,
   (...)
     90     val_range=(5, 1),
     91 ):
---> 92     return db.validate_dataset(
     93         self._df, paq_aliases, allow_lockdown, allow_na, verbose, val_range
     94     )

File ~\Documents\GitHub\Soundscapy\soundscapy\database.py:97, in validate_dataset(df, paq_aliases, allow_lockdown, allow_paq_na, verbose, val_range)
     95 if verbose > 0:
     96     print("Checking PAQ data quality.")
---> 97 if l := paq_data_quality(df, verbose, allow_lockdown, allow_paq_na, val_range):
     98     excl_df = df.iloc[l, :]
     99     df = df.drop(df.index[l])

File ~\Documents\GitHub\Soundscapy\soundscapy\database.py:134, in paq_data_quality(df, verbose, allow_lockdown, allow_na, val_range)
...
    142 
    143     """
--> 144     return db.return_paqs(df, incl_ids, other_cols)

NameError: name 'df' is not defined
```

**Comment by MitchellAcoustics** [comment]
Solved in the major refactor in #58
